test/fail/ok/bad-type-comp.tc.ok

```diff
@@ -2,10 +2,18 @@ bad-type-comp.mo:5.10-5.19: type error [M0096], expression of type
   module {}
 cannot produce expected type
   module {type T = Null}
+because: Unsupported additional field `T` in type 
+  module {type T = Null}
+ of top level
 bad-type-comp.mo:6.10-6.39: type error [M0096], expression of type
   module {type T = Int}
 cannot produce expected type
   module {type T = Null}
+because: The type 
```

Comment:
This one looks a bit weird. Maybe we need a better printer for standalone Typ c cases